src/styles/landingpage/middlepart.module.scss

```diff
@@ -0,0 +1,58 @@
+@import "variable";
+@import "mixins";
+
+.outside {
+  background-color: $seperationColor;
+  padding: 5px 0;
+}
+.generalContainer {
+  @include centerStuff;
+  background-color: #000;
+  color: white;
+  padding: 25px 60px;
+}
+.contentContainer {
+  width: 40%;
+  padding: 0 65px;
+  .title {
+    font-size: 3.5rem;
+  }
+  .subtitle {
+    font-size: 1.6rem;
+    font-weight: 400;
+  }
+}
+.videoContainer {
+  width: 48%;
+  height: 100%;
+  flex: 0 1 auto;
+  box-sizing: border-box;
+}
+.animationContainer {
+  position: relative;
+  overflow: hidden;
+}
+.tv {
+  max-width: 100%;
+  height: auto;
+  border: 0;
+  z-index: 100;
```

Comment:
if you use z-index please get the values from `variables`.  It's important to learn how to handle z-index in the project otherwise it's go out of control

[Recomended Article](https://css-tricks.com/handling-z-index/)
[Optional](https://medium.com/ccoutzoukis/better-z-index-management-with-sass-a8095ba3a2bc)